### PR TITLE
Bring snippets up to WordPress coding standards.

### PR DESCRIPTION
**Pull request by timothyjensen**
Just some minor formatting adjustments to bring the snippets inline with [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards).

### CODE REVIEW

File: options-and-settings-pages/genesis-cpt-archive-metabox.php [hunk 1]
```diff
@@ -92,30 +101,36 @@ public function hooks() {
...
-		// Include CMB CSS in the head to avoid FOUC
+		// Include CMB CSS in the head to avoid FOUC.
 		add_action( "admin_print_styles-{$this->admin_hook}", array( 'CMB2_hookup', 'enqueue_cmb_css' ) );
 
 		// Hook into the genesis cpt setttings save and add in the CMB2 sanitized values.
-		add_filter( "sanitize_option_genesis-cpt-archive-settings-{$this->post_type}", array( $this, 'add_sanitized_values' ), 999 );
+		add_filter( "sanitize_option_genesis-cpt-archive-settings-{$this->post_type}", array(
+			$this,
+			'add_sanitized_values',
+		), 999 );
```
Comment (jtsternberg) [inline options-and-settings-pages/genesis-cpt-archive-metabox.php]: not sure how the sniffer missed this, but callback arrays should not be split to multiple lines.

File: options-and-settings-pages/genesis-cpt-archive-metabox.php [hunk 2]
```diff
@@ -232,15 +253,15 @@ public function __get( $field ) {
...
  * @return Myprefix_Genesis_CPT_Settings_Metabox object
  */
 function myprefix_genesis_cpt_settings( $post_type ) {
 	return Myprefix_Genesis_CPT_Settings_Metabox::get_instance( $post_type );
 }
 
-// Get it started
-// myprefix_genesis_cpt_settings( 'custom-post-type-slug' );
+// Get it started.
+//myprefix_genesis_cpt_settings( 'custom-post-type-slug' );
```
Comment (jtsternberg) [inline options-and-settings-pages/genesis-cpt-archive-metabox.php]: I'm not a fan of `//comments w/o space`

Comment (timothyjensen) [inline options-and-settings-pages/genesis-cpt-archive-metabox.php]: Okay, I'll revert this change.

File: options-and-settings-pages/genesis-cpt-archive-metabox.php (hunk 1 above)
Comment (timothyjensen) [inline options-and-settings-pages/genesis-cpt-archive-metabox.php]: Yep, I'll revert this.

**Comment by jtsternberg** [comment]
Thank you sir!

**Comment by timothyjensen** [comment]
No, thank you!  This will be a great resource for Genesis folks.

**Pull request merged**